auth0/v3/management/organizations.py

```diff
@@ -31,21 +31,35 @@ def _url(self, *args):
         return url
 
     # Organizations
-    def all_organizations(self, page=None, per_page=None):
+    def all_organizations(self, page=None, per_page=None, include_totals=True, from_param=None, take=None):
         """Retrieves a list of all the organizations.
 
         Args:
-           page (int): The result's page number (zero based). When not set,
-              the default value is up to the server.
+            page (int): The result's page number (zero based). When not set,
+                the default value is up to the server.
 
-           per_page (int, optional): The amount of entries per page. When not set,
-              the default value is up to the server.
+            per_page (int, optional): The amount of entries per page. When not set,
+                the default value is up to the server.
+
+            include_totals (bool, optional): True if the query summary is
+                to be included in the result, False otherwise. Defaults to True.
+
+            from_param (str, optional): Checkpoint Id form which to begin retrieving results.
+                You can limit the amount of logs using the take parameter.
```

Comment:
Thought of using "organizations" but it's easier to copy/paste this again and keep making that mistake.
```suggestion
                You can limit the number of entries using the take parameter.
```